orchestration/src/test/java/com/sap/ai/sdk/orchestration/client/OrchestrationUnitTest.java

```diff
@@ -116,73 +152,69 @@ void setup(WireMockRuntimeInfo server) {
-            .inputParams(inputParams);
-
-    final var result = client.orchestrationV1EndpointsCreate(config);
-
-    assertThat(result.getRequestId()).isEqualTo("26ea36b5-c196-4806-a9a6-a686f0c6ad91");
-    assertThat(result.getModuleResults().getTemplating().get(0).getContent())
-        .isEqualTo("Reply with 'Orchestration Service is working!' in German");
-    assertThat(result.getModuleResults().getTemplating().get(0).getRole()).isEqualTo("user");
-    var llm = result.getModuleResults().getLlm();
-    assertThat(llm.getId()).isEqualTo("chatcmpl-9lzPV4kLrXjFckOp2yY454wksWBoj");
-    assertThat(llm.getObject()).isEqualTo("chat.completion");
-    assertThat(llm.getCreated()).isEqualTo(1721224505);
-    assertThat(llm.getModel()).isEqualTo("gpt-35-turbo-16k");
-    var choices = llm.getChoices();
-    assertThat(choices.get(0).getIndex()).isEqualTo(0);
-    assertThat(choices.get(0).getMessage().getContent())
-        .isEqualTo("Orchestration Service funktioniert!");
-    assertThat(choices.get(0).getMessage().getRole()).isEqualTo("assistant");
-    assertThat(choices.get(0).getFinishReason()).isEqualTo("stop");
-    var usage = llm.getUsage();
-    assertThat(usage.getCompletionTokens()).isEqualTo(7);
-    assertThat(usage.getPromptTokens()).isEqualTo(19);
-    assertThat(usage.getTotalTokens()).isEqualTo(26);
-    assertThat(result.getOrchestrationResult().getId())
-        .isEqualTo("chatcmpl-9lzPV4kLrXjFckOp2yY454wksWBoj");
-    assertThat(result.getOrchestrationResult().getObject()).isEqualTo("chat.completion");
-    assertThat(result.getOrchestrationResult().getCreated()).isEqualTo(1721224505);
-    assertThat(result.getOrchestrationResult().getModel()).isEqualTo("gpt-35-turbo-16k");
-    choices = result.getOrchestrationResult().getChoices();
-    assertThat(choices.get(0).getIndex()).isEqualTo(0);
-    assertThat(choices.get(0).getMessage().getContent())
-        .isEqualTo("Orchestration Service funktioniert!");
-    assertThat(choices.get(0).getMessage().getRole()).isEqualTo("assistant");
-    assertThat(choices.get(0).getFinishReason()).isEqualTo("stop");
-    usage = result.getOrchestrationResult().getUsage();
-    assertThat(usage.getCompletionTokens()).isEqualTo(7);
-    assertThat(usage.getPromptTokens()).isEqualTo(19);
-    assertThat(usage.getTotalTokens()).isEqualTo(26);
-
-    // verify that null fields are absent from the sent request
-    final String request =
-        new String(
-            getClass()
-                .getClassLoader()
-                .getResourceAsStream("templatingRequest.json")
-                .readAllBytes());
-    verify(postRequestedFor(urlPathEqualTo("/completion")).withRequestBody(equalToJson(request)));
+  void templating() throws IOException {
+    try (var inputStream = TEST_FILE_LOADER.apply("templatingResponse.json")) {
+      final String response = new String(inputStream.readAllBytes());
+
+      stubFor(post(urlPathEqualTo("/completion")).willReturn(okJson(response)));
```

Comment:
(Minor)

Wiremock allows for file lookups natively. Why not use that instead of `InputStream -> String`?

Reply:
Used wiremock reading files